Gruntfile.js

```diff
@@ -520,7 +536,8 @@ module.exports = function (grunt) {
          'npm-install', */
         'cleanempty',
         'exec:clean-nls',
-        'usemin'
+        'usemin',
+        'githooks'
```

Comment:
This seems wrong to me.  We'd want this to run automatically, not just when you manually run `build`.  That is, when I `commit`, or maybe even when I just stage changes, it should run.  There should be no manual intervention required, or people will forget/not-know to do it.